ts/components/session/SessionPasswordModal.tsx

```diff
@@ -31,6 +31,17 @@ export class SessionPasswordModal extends React.Component<Props, State> {
 
     this.setPassword = this.setPassword.bind(this);
     this.closeDialog = this.closeDialog.bind(this);
+
+    this.onKeyUp = this.onKeyUp.bind(this);
+    window.addEventListener('keyup', this.onKeyUp);
```

Comment:
No need for a global window eventlistener, we are only interested in events in the password inputs

Reply:
All ``.tsx`` files store event listeners globally.